packages/astro/src/default/utils/content.ts

```diff
@@ -272,21 +282,6 @@ export async function getTutorial(): Promise<Tutorial> {
   return _tutorial;
 }
 
-function pick<T extends Record<any, any>>(objects: (T | undefined)[], properties: (keyof T)[]) {
```

Comment:
This is replaced by `squash` and `inheritProperties` which perform a deeper inheritance. This was something we wanted for the `terminal` property but never got to implement. It was needed for the `i18n` property as otherwise you couldn't really override a single property for a chapter or lesson.